build.gradle

```diff
@@ -36,11 +36,13 @@ configure(subprojects.findAll { it.projectDir.toPath().resolve('sql').toFile().e
 
   // For all Node subprojects
   sp.plugins.withId('com.github.node-gradle.node') {
-    // Distribute SQL as part of resource processing
-    tasks.create('processResources')
-    tasks.findAll { it.name == 'processResources' }.each { it.dependsOn distributeSql }
-    // Remove distributed SQL as part of `clean` tasks
-    // 'clean' is not visible here, use the subproject .gradle 'clean' task
+    if (!sp.projectDir.toString().contains('scheduler-server')) { // scheduler-server is both a java and a node project
```

Comment:
Hmm does `if (!sp.plugins.hasPlugin('java'))` not work here?

Reply:
Didn't work for me 😅  Happy to be proven wrong. I think it has to do with the order with which the build.gradle files are evaluated - the top level file gets evaluated before the subprojects do, so hasPlugin returns false... actually I'm not sure that's correct either. 🤷

Reply:
Gotcha. I'll play around with this for a minute, otherwise what you have will have to do. Thanks

Reply:
Created https://github.com/NASA-AMMOS/aerie/pull/77/files to generalize this.